src/app/(affiliate)/affiliate/events/page.tsx

```
@@ -254,160 +178,165 @@
-                              <div className="flex items-start justify-between">
-                                <div className="space-y-2">
-                                  <div className="flex items-center gap-2">
-                                    <CardTitle className="text-lg">{event.title}</CardTitle>
-                                    <Badge className={getStatusColor(event.status)}>
-                                      {event.status}
-                                    </Badge>
-                                    <Badge variant="outline">{event.category}</Badge>
+                  <div className="space-y-4">
+                    <div className="grid gap-4">
+                      {performanceSummary.map((event) => (
+                        <Card className="shadow-md">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)